pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -753,12 +754,12 @@ func (oe *operationExecutor) VerifyVolumesAreAttached(
 				volumeSpecMapByPlugin[pluginName] = volumeSpecMap
 				continue
 			}
-
 			// If node doesn't support Bulk volume polling it is best to poll individually
-			nodeError := oe.VerifyVolumesAreAttachedPerNode(nodeAttachedVolumes, node, actualStateOfWorld)
-			if nodeError != nil {
-				klog.Errorf("VerifyVolumesAreAttached failed for volumes %v, node %q with error %v", nodeAttachedVolumes, node, nodeError)
-			}
+			needVerifyVolumes = append(needVerifyVolumes, volumeAttached)
```

Comment:
@Pingan2017 Good Catch! Thank you for fixing it!

Just a nit, how about needIndividualVerifyVolumes to make the name even more clear?

Reply:
@jingxu97  Thanks for your suggestion！
I have fixed it.